integration-tests/bats/helper/query-server-common.bash

```diff
@@ -256,86 +200,33 @@ stop_sql_server() {
     SERVER_PID=
 }
 
-# server_query connects to a running mysql server, executes a query and compares the results against what is expected.
-# In the event that the results do not match expectations, the python process will exit with an exit code of 1
+# server_query connects to a running mysql server,
+# executes a query (or list of queries separated by a `;`),
+# and compares the results against what is expected.
+#
+# If executing multiple queries,  separate the expected CSV values with a `;`.
+#
+# If you expect an exception, leave query results blank and add an additional
```

Comment:
i like the expanded docstring, only thing i might add is example commands corresponding to the features you point out